application/src/main/java/com/musicforall/services/DbPopulateService.java

```diff
@@ -117,6 +125,19 @@ private void populate() {
                 .peek(u -> LOG.info("going to save file by url - {}", u))
                 .map(url -> (Callable<Path>) () -> fileManager.save(url))
                 .collect(toList());
+
+        final int FOLLOWED_USER_ID = 2;
+        final int TRACK_1_ID = 1;
+        final int TRACK_2_ID = 2;
+        final int TRACK_3_ID = 3;
+
+        historyEventListener.handleTrackLiked(new TrackLikedEvent(TRACK_1_ID, FOLLOWED_USER_ID));
+        historyEventListener.handleTrackLiked(new TrackLikedEvent(TRACK_1_ID, FOLLOWED_USER_ID));
+        historyEventListener.handleTrackLiked(new TrackLikedEvent(TRACK_3_ID, FOLLOWED_USER_ID));
+        historyEventListener.handleTrackLiked(new TrackLikedEvent(TRACK_2_ID, FOLLOWED_USER_ID));
+        historyEventListener.handleTrackLiked(new TrackLikedEvent(TRACK_2_ID, FOLLOWED_USER_ID));
+        historyEventListener.handleTrackLiked(new TrackLikedEvent(TRACK_2_ID, FOLLOWED_USER_ID));
+
```

Comment:
move filling db with test data from main app to history module like for track listening.